### ISSUE TITLE
int to RLP mismatching with RLP to int

### ISSUE THREAD

**Issue by jolestar** [walleth/kethereum#49]
```kotlin
import org.kethereum.functions.rlp.toIntFromRLP
import org.kethereum.functions.rlp.toRLP

class RLPTest{

    @Test
    fun testIntRLP(){
        val int = 1838383984
        val rlp = int.toRLP()
        println(rlp.bytes.size)
        val int1 = rlp.toIntFromRLP()
        Assert.assertEquals(int, int1)
    }
}
```
output:
```text
4

java.lang.AssertionError: 
Expected :1838383984
Actual   :1821541232
```

**Comment by ligi** [comment]
Thanks for the report! Will try to have a look today

**Comment by ligi** [comment]
I can confirm this bug and have a fix. Will add your int to the test-vectors. Thanks so much for reporting!!

**Comment by ligi** [comment]
Fixed in v0.66 - thanks so much for the quality bug report - love reports that contain a test!
